src/pages/Catagories.js

```diff
@@ -0,0 +1,9 @@
+import classes from './Catagories.module.css';
+
+const Catagories = () => (
+  <div className={classes.catagories}>
+    <h1>Catagories Page</h1>
+  </div>
```

Comment:
- [ ] According to the requirements this page should have only a button with the text "Check status". kindly make this change.